sbom.cdx.json

```diff
@@ -0,0 +1,38 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "version": 1,
+  "metadata": {
+    "authors": [
+      {
+        "name": "@VCS_SBOM_AUTHORS@"
+      }
+    ]
+  },
+  "components": [
+    {
+      "type": "library",
+      "bom-ref": "pkg:github/libtom/libtomcrypt@@VCS_TAG@",
+      "cpe": "cpe:2.3:a:libtom:libtomcrypt:@VCS_TAG@:*:*:*:*:*:*:*",
+      "name": "libtomcrypt",
+      "version": "@VCS_VERSION@",
+      "description": "Modular and portable cryptographic toolkit",
+      "supplier": {
+        "name": "libtomcrypt developers"
+      },
```

Comment:
@libtom/libtomcrypt, FYI: if "supplier" feels uncomfortable, the other option is to have this instead:

``` json
      "authors": {
        "name": "libtomcrypt developers"
      },
```

Alternatively, all the authors can be enumerated:

``` json
      "authors": {
        "name": "@VCS_AUTHORS@"
      },
```

Reply:
Yes, absolutely. I'm happy to fix up the PR with either option.

Reply:
I guess option 1 would be my preferred way to go.

And I like the idea of having this file in a `contrib` folder.

Reply:
Understood, thanks for the feedback. I've updated the PR with option 1 and moved it into a new contrib folder.